code/game/objects/effects/spawners/random/structure.dm

```diff
@@ -6,14 +6,14 @@
 	name = "crate spawner"
 	icon_state = "crate"
 	loot = list(
-		/obj/effect/spawner/random/structure/crate_empty = 75,
-		/obj/structure/closet/crate/trashcart/filled = 7,
-		/obj/effect/spawner/random/trash/moisture_trap = 5,
-		/obj/effect/spawner/random/trash/hobo_squat = 3,
-		/obj/structure/closet/mini_fridge = 3,
-		/obj/item/kirbyplants/random = 3,
-		/obj/effect/spawner/random/trash/mess = 3,
-		/obj/structure/closet/crate/decorations = 1,
+		/obj/effect/spawner/random/structure/crate_loot = 745,
```

Comment:
There is already a `crate_loot` random spawner that is used for all maps.  This crate spawner is supposed to be empty.

Reply:
It used to spawn full crates yesterday.

Reply:
Actually, after looking at the old code it seems the original used the `crate_loot` and I thought I had replaced the crates with it but it was actually for a different spawner that spawned `crate_loot` at a 20% chance.  This is fine then, but keep in mind that the spawner for `crate_loot` isn't a 100% chance.

Reply:
i get the feeling something went wrong it still doesn't appear in maints